SocialFilter/sections/specific.txt

```diff
@@ -72,6 +72,8 @@ antaranews.com,jn.pt,insider.com,spur.hpplus.jp,yuru-mbti.com,story.hr,gihyo.jp,
 !
+ilsecoloxix.it##.story__share
```

Comment:
This component is responsible for opening a chat, which is a function button. I don't think it's worth blocking this block, because social buttons are already blocked, only email and chat are left, which can be useful for users.

Reply:
although there is a separate section for comments

Reply:
Oh ok :)